SimPEG/Survey.py

```diff
@@ -20,7 +22,7 @@ class BaseRx(object):
 
     def __init__(self, locs, rxType, **kwargs):
         self.uid = str(uuid.uuid4())
-        self.locs = locs
+        self.locs = np.atleast_2d(locs)
```

Comment:
is this okay for DC case (e.g. dipole Rx)?

Reply:
Under the hood, we always assume that the receiver locations are input as a 2D array. This takes care of the case where the user inputs a 1D array, e.g. `np.r_[0., 0., 10.]` and casts it to a 2D array. 

With respect to the DC code - it does mean that the `locs` which were a list are now a 3D numpy array. However, this doesn't change any behavior. Asking for `rx.locs[0]` will still return the 2D array of M-electrode locations, and `rx.locs[1]` will return the N-electrode locations as expected. I tested this on http://docs.simpeg.xyz/content/examples/06-dc/plot_dipoledipole_2Dinversion_twocylinders.html 

![image](https://user-images.githubusercontent.com/6361812/42530811-fc18fe38-8436-11e8-8465-4faa18e6d90b.png)

Reply:
nice! thanks @lheagy